tests/UI/pages/FO/checkout/index.js

```diff
@@ -245,6 +250,25 @@ class Checkout extends FOBasePage {
     await this.clickAndWaitForNavigation(page, this.paymentConfirmationButton);
   }
 
+  /**
+   * Get All tax included price
+   * @param page {Page} Browser tab
+   * @returns {Promise<number>}
+   */
+  getATIPrice(page) {
```

Comment:
```suggestion
  getAllTaxesIincludedPrice(page) {
```

Reply:
Do you suggest updating the function name 'getATIPrice' for all Pages/Scenarios? 
Or duplicating the existing one to create one specific, with name 'getAllTaxesIincludedPrice' to my use case?

Reply:
Oops sorry. I didn't know it was an override. I close my comment.